### DISCUSSION TITLE
Room

### DISCUSSION THREAD

**Discussion by Jsueeee** [euphony-io/check-in-listener#7] [General]
@eunzinn 
은진님! 우리 룸 같이 하기로 했죠!
Room의 기본은 database, dao, entity입니다
일단 처음이시니까 dao랑 entity 자료 참고해보셔서 작성 한 번 해보실 수 있으실까요??

https://developer.android.com/training/data-storage/room/defining-data?hl=ko
https://developer.android.com/training/data-storage/room/accessing-data?hl=ko

일단 공식문서 링크 남기는데, 그냥 구글에 Room 이라고 검색하셔서 블로그 글 보셔도 됩니다!

**Comment by eunzinn** [comment]
어제 강의 보면서 dao랑 entity 파일 만들고 정말 짧게 몇 줄 적어봤는데 제대로 한 건지 봐주실 수 있을까요??
이 다음부터 MainActivity에서 room 접근하는 걸 작성해야하는게 맞는지 궁금합니다!

**Reply by Jsueeee** [reply]
넵 당연하죠! 올려주세요!!

**Comment by eunzinn** [comment]
#9  감사합니다 ㅎㅎ PR 올렸습니다!

**Comment by Jsueeee** [comment]
@eunzinn 룸 좀 수정 + 데이터베이스 추가해서 머지해놨으니 참고하시면 좋을 것 같습니다!
